test/Microsoft.Azure.WebJobs.Host.FunctionalTests/JobHostOptionsTests.cs

```diff
@@ -8,52 +8,51 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
-using Moq;
 using Newtonsoft.Json;
 using Xunit;
 
 namespace Microsoft.Azure.WebJobs.Host.UnitTests
 {
     public class JobHostOptionsTests
     {
-        // TODO: DI: Change to use IHostingEnvironment
-        //[Theory]
-        //[InlineData(null, false)]
-        //[InlineData("Blah", false)]
-        //[InlineData("Development", true)]
-        //[InlineData("development", true)]
-        //public void IsDevelopment_ReturnsCorrectValue(string settingValue, bool expected)
-        //{
-        //    using (EnvVarHolder.Set(Constants.EnvironmentSettingName, settingValue))
-        //    {
-        //        JobHostOptions config = new JobHostOptions();
-        //        Assert.Equal(config.IsDevelopment, expected);
-        //    }
-        //}
-
-        //public void UseDevelopmentSettings_ConfiguresCorrectValues()
-        //{
-        //    using (EnvVarHolder.Set(Constants.EnvironmentSettingName, "Development"))
-        //    {
-        //        JobHostOptions config = new JobHostOptions();
-        //        Assert.False(config.UsingDevelopmentSettings);
-
-        //        if (config.IsDevelopment)
-        //        {
-        //            config.UseDevelopmentSettings();
-        //        }
-
-        //        Assert.True(config.UsingDevelopmentSettings);
-        //        Assert.Equal(TimeSpan.FromSeconds(2), config.Queues.MaxPollingInterval);
-        //        Assert.Equal(TimeSpan.FromSeconds(15), config.Singleton.ListenerLockPeriod);
-        //    }
-        //}
+        // TODO: DI: private readonly Change to private readonly use IHostingEnvironment
+        [Theory(Skip = "Development settings not being applied")]
+        [InlineData(null, false)]
+        [InlineData("Blah", false)]
+        [InlineData("Development", true)]
+        [InlineData("development", true)]
+        public void IsDevelopment_ReturnsCorrectValue(string settingValue, bool expected)
+        {
+            //using (EnvVarHolder.Set(Constants.EnvironmentSettingName, settingValue))
+            //{
+            //    JobHostOptions config = new JobHostOptions();
+            //    Assert.Equal(config.IsDevelopment, expected);
+            //}
+        }
+
+        [Fact(Skip = "Development settings not being applied")]
+        public void UseDevelopmentSettings_ConfiguresCorrectValues()
+        {
+            //using (EnvVarHolder.Set(Constants.EnvironmentSettingName, "Development"))
+            //{
+            //    JobHostOptions config = new JobHostOptions();
+            //    Assert.False(config.UsingDevelopmentSettings);
+
+            //    if (config.IsDevelopment)
+            //    {
+            //        config.UseDevelopmentSettings();
+            //    }
+
+            //    Assert.True(config.UsingDevelopmentSettings);
+            //    Assert.Equal(TimeSpan.FromSeconds(2), config.Queues.MaxPollingInterval);
```

Comment:
Tracked by Tracked by https://github.com/Azure/azure-webjobs-sdk/issues/1869. Add a link to that bug